README.md

```diff
@@ -1 +1,100 @@
-# React-capston
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Awesome books ](#-weather-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Weather <a name="about-project"></a>
+
+**Weather App** in this project, i create a weather app using React.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[Search your city]**
+- **[Go to detail page]**
+- **[React Router Dom]**
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML | CSS | JS | ES6 | React
+
+- More details about the project can be seen in this [video](https://www.loom.com/share/2d54f1e084ca4ed388b09794ed6a21a7) 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [live link](https://weather-tcql.onrender.com)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A GitHub | A code editor | A web browser
+
+### Install
+
+Install all project dependencies by running the command below
+
+  `$ npm install`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  `git clone https://github.com/huzaifakhan0308/react-capston`
+
+### Usage
+
+To run the project, run `npm start` to run the project
+
+## 👥 Author <a name="author"></a>
+
+👤 **Huzaifa khan**
+
+- GitHub: [@huzaifakhan0308](https://github.com/huzaifakhan0308)
+- Twitter: [@huzaifakhan0308](https://twitter.com/home?lang=en)
+- LinkedIn: [huzaifa khan](https://www.linkedin.com/in/huzaifa-khan-938140256/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[setting page]**
+- **[Add more styles]**
+- **[login & sign up]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check [issues page](https://github.com/huzaifakhan0308/react-capston/issues).
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)for this project, "`The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.
`", please kindly acknowledge [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) for his original design idea.